tests/resources/case_title_form_loading/suite.xml

```diff
@@ -53,6 +53,24 @@
         </push>
       </stack>
     </action>
+    <action>
+      <display>
+        <text>
+          <locale id="case_list_form.m0"/>
+        </text>
+        <text form="image">
+          <locale id="case_list_form.m0.icon"/>
+        </text>
+      </display>
+      <stack>
+        <push>
+          <command value="'m0-f0'"/>
```

Comment:
Needed to push to a command that requires a nodeset, since otherwise getNeededData() will be null in the old (incorrect) loop predicate and would still work correctly